pkg/services/grafana.go

```diff
@@ -17,9 +17,9 @@ import (
 )
 
 type GrafanaOptions struct {
-	ApiUrl             string `json:"apiUrl"`
-	ApiKey             string `json:"apiKey"`
-	InsecureSkipVerify bool   `json:"insecureSkipVerify"`
+	ApiUrl    string                         `json:"apiUrl"`
+	ApiKey    string                         `json:"apiKey"`
+	Transport httputil.HTTPTransportSettings `json:"transport"`
```

Comment:
Same as with `AlertmanagerOptions`, we cannot break backwards compatibility.

Reply:
plz correct me, should it be sth like this and pass it through httputil.HTTPTransportSettings?
```
type GrafanaOptions struct {
	ApiUrl             string `json:"apiUrl"`
	ApiKey             string `json:"apiKey"`
	InsecureSkipVerify bool   `json:"insecureSkipVerify"`
	MaxIdleConns       int    `json:"maxIdleConns"`
	MaxIdleConnsPerHost int   `json:"maxIdleConnsPerHost"`
	MaxConnsPerHost    int    `json:"maxConnsPerHost"`
	IdleConnTimeout    time.Duration `json:"idleConnTimeout"`
}
```

Reply:
@afzal442 yes that looks good, although we wouldn't really need an `httputil.HTTPTransportSettings` if we were to do it in that way.